### DISCUSSION TITLE
Why does plugin cause i3 scratchpad terminal to disappear?

### DISCUSSION THREAD

**Discussion by mashtonian** [PatrickF1/fzf.fish#278] [Q&A]
Bit of an odd one:

When I use the plugin to select a file, in an i3 scratchpad alacritty terminal, when I press Enter to choose the selected file then terminal moves out to the scratchpad, i.e it disappears off screen. If I then press the i3 keybinding to bring the terminal back to the screen it has the selected file in the prompt as expected.

Why would the plugin cause the terminal to move back to the scratchpad?

**Comment by PatrickF1** [comment]
What's this scratchpad you're referring to?

**Comment by mashtonian** [comment]
It's a feature in the i3 window manager, whereby windows can be moved off screen to the "scratchpad" (kind of hidden, sort of like an equivalent of being minimised in Windows).

I have a keyboard shortcut defined (Super+X) that moves an alacritty / fish terminal window to and from the scratchpad. This is configured in the i3 config file. So far, so irrelevant to your plugin.

The odd thing is that when I use the ctrl-alt-f (for example) plugin command and search for a file, I press enter to select it and the terminal windows disappears, off to the scratchpad. I can bring it back with Super-x. The selected file name and path are at the prompt as expected.

**Reply by PatrickF1** [reply]
That's odd. I don't think either fish shell nor fzf have the power to move your window to scratchpad. I am suspecting some sort of key bindings issues somehow i3 is reading Enter and a different key combo. Have you tried changing the shortcuts and seeing if the issue still happens. Sorry, but I can guarantee you this isn't an fzf.fish issue and as a non-Linux user, I don't think I can help here.
